swan-lake/resources/featured-scenarios/manage-data-persistence-with-bal-persist.md

```diff
@@ -172,7 +172,7 @@ final store:Client sClient = check new();
 
 public function main() returns error? {
    store:EmployeeInsert employee1 = {
-       id: uuid:createType4AsString(),
+       id: "16c6553a-373c-4b29-b1c8-c282f444248c",
```

Comment:
shall we remove uuid import in line no 166

Reply:
Removed.